modules/virt-pausing-vm-web.adoc

```diff
@@ -19,15 +19,15 @@ You can pause a virtual machine (VM) from the web console.
 * To stay on this page, where you can perform actions on multiple VMs:
 
 .. Click the Options menu {kebab} located at the far right end of the row and click *Pause VirtualMachine*.
+.. If action confirmation is enabled, in the confirmation dialog, click *Pause*.
```

Comment:
@sbeskin-redhat I think technically, all of these need to be reworked. From a programmatic perspective, the way you have it, you click Pause only if action confirmation is enabled, right? 

```suggestion
.. Click the Options menu {kebab} located at the far right end of the row and click *Pause VirtualMachine*.  If action confirmation is enabled a confirmation dialog opens.
.. Click *Pause* in the confirmation dialog.
```